### DISCUSSION TITLE
Player Kits - Name Pages

### DISCUSSION THREAD

**Discussion by SlyRiddles** [Zrips/CMI#9279] [Feature Requests]
Hello, I was wondering where it says pages 1>2>3> etc 
At the top left corner, it says Kits, I suggest a feature to allow pages to have specific names. Example

Kits > member kits
Page 1 = Member kits 

Admin Kits
Page = 2 Member kits

Commands like 
/cmi kiteditor page (#) rename (name)

**Comment by Zrips** [comment]
You can check locale file under titlesByPage section which allows you to define custom title names based on page number
